gui-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -60,20 +65,21 @@ BDEPEND="
 "
 
 src_configure() {
+	local backends="$(usex drm 'drm' '')"
+	backends+="$(usex libinput ',libinput' '')"
+	backends+="$(usex x11-backend ',x11' '')"
+	# Get rid of leading comma
+	backends="$(echo ${backends} | sed -e 's/^,//')"
```

Comment:
I'd like for this to avoid string manipulation if possible, but I don't see an easy way without crazy logic.

Reply:
I thought exactly the same, though no other obvious solutions come to my mind...

Reply:
Done. See https://github.com/gentoo/gentoo/pull/31029#discussion_r1198777949